factcast-store-pgsql/src/main/resources/db/changelog/factcast/issue1435/stats_index.sql

```diff
@@ -0,0 +1,17 @@
+CREATE VIEW stats_index AS
+SELECT t.schemaname,
+       t.tablename,
+       c.reltuples::bigint                            AS num_rows,
+       pg_size_pretty(pg_relation_size(c.oid))        AS table_size,
+       psai.indexrelname                              AS index_name,
+       pg_size_pretty(pg_relation_size(i.indexrelid)) AS index_size,
+       CASE WHEN i.indisunique THEN 'Y' ELSE 'N' END  AS "unique",
+       psai.idx_scan                                  AS number_of_scans,
+       psai.idx_tup_read                              AS tuples_read,
+       psai.idx_tup_fetch                             AS tuples_fetched
+FROM pg_tables t
```

Comment:
Call me crazy but I would love to have `i.indisvalid` here as well 🥸

Reply:
plz recheck